src/components/error/error.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import cn from 'classnames';
+
+ type Props = {
+   error: string;
+   closeError: () => void;
+ };
+
+export const Error: React.FC<Props> = ({ error, closeError }) => {
+  const [isVisible, setIsVisible] = useState(true);
+
+  setTimeout(() => {
```

Comment:
You should clear all timers on the component unmount to avoid unpredictable side effects https://bobbyhadz.com/blog/react-cleartimeout